include/seqan3/argument_parser/auxiliary.hpp

```diff
@@ -248,6 +248,13 @@ enum option_spec
                    */
 };
 
+//!\brief Indicates whether application allows automatic update notifications by the seqan3::argument_parser.
+enum class update_notifications
+{
+    enabled, //!< Automatic update notifications should be enabled.
+    disabled //!< Automatic update notifications should be disabled.
+};
```

Comment:
```suggestion
    on, //!< Automatic update notifications should be enabled.
    off //!< Automatic update notifications should be disabled.
};
```

Just wondering, as it is essentially the same but shorter.